core/jobs/transforms/validation/base_validation_test.py

```diff
@@ -345,9 +345,8 @@ def _get_model_domain_object_instance(
         """
         return MockDomainObject()
 
-    # We have ignored [override] here because the signature of this method
-    # doesn't match with super class's _get_domain_object_validation_type
-    # method.
+    # Here we use MyPy ignore because the signature of this method doesn't
```

Comment:
Probably should expain why we can't make it match?

Reply:
Comment updated